snackbar/material/task.md

```diff
@@ -0,0 +1,69 @@
+---
+pdf:
+  title: 'Inf-Einf-B Übung: Kiosk'
+---
+
+# Kiosk
+
+> Diese Aufgabe ist eine abgewandelte Version des [Snackbar Practice Problems von CS50](https://cs50.harvard.edu/x/2024/practice/snackbar/).
+>
+> **Disclaimer:** Diese Aufgabe wurde nicht vom Lehrstuhl herausgegeben und kann Fehler enthalten. Sie dient lediglich zu Übungszwecken!
+
+## Teilaufgabe 1
+
+Stell dir vor, du bist am Strand und möchtest eine Reihe von Gerichten am Kiosk bestellen. Du hast nur eine begrenzte Menge Bargeld bei dir und möchtest vor der Bestellung den Gesamtpreis deiner Gerichte wissen.
+
+Wir sind Informatiker! Somit würden wir ein solches Problem grundsätzlich mithilfe eines Algorithmus lösen. Implementiere diesen in einer C-Datei namens `snackbar.c`. In `snackbar.c` wirst du unter anderem zwei Funktionen und eine `struct` implementieren. 
```

Comment:
I'd rather move the introduction (everything besides "Implementiere diesen ...") up a heading level to "Kiosk" as the introduction is somewhat independent from the actual subtask.